build/regressions.config.yml

```diff
@@ -39,6 +39,28 @@ sources:
         - sebastienros
         - agocke
       template: build
+  - name: MVC 
+    table: MvcBenchmarks
+    rules:
+      - include: ApiCrud
```

Comment:
@sebastienros 
Is it correct to assume that these are prefixes and that indicate the list of `scenarios` to include?

Reply:
These are regular expression that will match the scenario and description arguments used in the command line. In this case you can just use a "." catch all regex that will be applied to everything in this table. It's useful for instance to add a different rule that can exclude some specific scenarios, or to add custom labels/owners. For instance you could only tag yourself for anything that has "Api.*" (just an example).

It makes me realize that the command line arguments in the azdo yaml file needs to be updated to this:

```
"args": [ "${{ s.arguments }} $(azureProfile) --profile ${{ parameters.profile }} --application.channel edge --application.framework net5.0 --variable warmup=${{ parameters.warmup }} --variable duration=${{ parameters.duration }} --application.options.collectCounters true --no-metadata --no-measurements --session $(session) --description \"${{ s.displayName }} ${{ parameters.profile }}\" --property buildId=\"$(buildId)\" --property buildNumber=\"$(buildNumber)\" --table MvcBenchmarks --sql SQL_CONNECTION_STRING --chart" ]
```

Reply:
I'm not sure where I need to update this command